content/800-guides/330-github-actions.mdx

```diff
@@ -0,0 +1,583 @@
+
+This is useful when testing database changes in isolation or when each feature branch needs its own temporary environment.
+
+## Prerequisites
+
+Make sure you have the following:
+
+- Node.js 18 or later
+- A [Prisma Data Platform](https://console.prisma.io?utm_source=actions_guide&utm_medium=docs) account
+- GitHub repository
+
+## 1. Set up your project
+
+Initialize your project:
+
+```bash
+mkdir prisma-gha-demo && cd prisma-gha-demo
+npm init -y
+```
+
+## 2. Install and Configure Prisma
+
+### 2.1. Install dependencies
+
+To get started with Prisma, install the required dependencies:
+
+```terminal
+npm install prisma tsx @prisma/client dotenv --save-dev
+npm install @prisma/extension-accelerate @prisma/client
+```
+
+Once installed, initialize Prisma:
+
+```bash
+npx prisma init --db --output ../src/generated/prisma
+```
+
+This creates:
+
+* A `prisma/` directory with `schema.prisma`
+* A `.env` file with `DATABASE_URL`
+* A generated client in `src/generated/prisma`
+
+### 2.2. Define your Prisma schema
+
+Edit `prisma/schema.prisma` to:
+
+```prisma file=prisma/schema.prisma
+generator client {
+  provider = "prisma-client"
+  output   = "../src/generated/prisma"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+  posts Post[]
+}
+
+model Post {
+  id        Int     @id @default(autoincrement())
+  title     String
+  content   String?
+  published Boolean @default(false)
+  authorId  Int
+  author    User    @relation(fields: [authorId], references: [id])
+}
+```
+
+### 2.3. Run initial migration and generate client
+
+```bash
+npx prisma migrate dev --name init
+```
+
+This pushes your schema and prepares the client.
+
+### 2.4. Seed the database
+
+Create a file at `src/seed.ts`:
+
+```tsx file=src/seed.ts
+import { withAccelerate } from "@prisma/extension-accelerate";
+import { PrismaClient } from "../src/generated/prisma/client";
+import "dotenv/config";
+
+const prisma = new PrismaClient().$extends(withAccelerate());
+
+const userData = [
+  {
+    name: "Alice",
+    email: "alice@prisma.io",
+    posts: {
+      create: [
+        {
+          title: "Join the Prisma Discord",
+          content: "https://pris.ly/discord",
+          published: true,
+        },
+        {
+          title: "Prisma on YouTube",
+          content: "https://pris.ly/youtube",
+        },
+      ],
+    },
+  },
+  {
+    name: "Bob",
+    email: "bob@prisma.io",
+    posts: {
+      create: [
+        {
+          title: "Follow Prisma on Twitter",
+          content: "https://twitter.com/prisma",
+          published: true,
+        },
+      ],
+    },
+  },
+];
+
+export async function main() {
+  for (const u of userData) {
+    await prisma.user.create({ data: u });
```

Comment:
I get an error that `user` is not defined.

<img width="853" height="359" alt="Screenshot 2025-07-14 at 9 34 28 PM" src="https://github.com/user-attachments/assets/14b31876-1281-44e0-8af5-9dcac351322a" />